### DISCUSSION TITLE
Is all the Special_Configurations files moved?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#1140] [General]
For some month ago in the dir (https://github.com/mriscoc/Special_Configurations) i found a lot config files, now its gone.
configuration.h and configuration_adv.h for ex. Ender3S1-F4-LA is gone?

**Comment by mriscoc** [comment]
Some time ago, the static configuration files were replaced by the Configurator python tool.

**Comment by gitfve** [comment]
You made pre build for the Ender3S1-F4-UBL

If i do the same by the config tools i need manually take some things out, because of the F4_CPU.
What i try to say, we are missing the ./Features/FreeFlash.json (or another fixer_file)

Dont know if you understand what i ask for...

**Reply by mriscoc** [reply]
FreeFlash.json is a custom json where you can disable features to make room in program memory for other features. Also you can manually disable features by editing the configuration files generated.

**Comment by gitfve** [comment]
Yes i understand that... What i missing is the possibility to compile the fw for my Ender3S1-F4-LA. Same as the file you release as a bin file.
But i see you have removed a lot for the fw to fit.

If you dont understand what i ask for i can explain better.
